assets/js/jquery-3.6.0.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
would be better to use a cdn rather than hosting this ourselves - e.g. like in DPS 
 https://github.com/ministryofjustice/digital-prison-services/blob/b752f6e474f6114c35c4eb0d6ca2d6bc07b60809/views/partials/layout.njk#L29

Reply:
used XMLHttpRequest so not an issue now 🙂